db/migrate/20150502102511_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,22 @@
+class AddConfirmableToDevise < ActiveRecord::Migration
+  # Adding according to Devise docs
+  # Ref: https://github.com/plataformatec/devise/wiki/How-To:-Add-:confirmable-to-Users
+  # /dwessman
+  def up
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_column :users, :unconfirmed_email, :string # Only if using reconfirmable
+    add_index :users, :confirmation_token, unique: true
+    # User.reset_column_information # Need for some types of updates, but not for update_all.
+    # To avoid a short time window between running the migration and updating all existing
+    # users as confirmed, do the following
+    execute("UPDATE users SET confirmed_at = NOW()")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.